components/UI/Map/LeafletMap.tsx

```diff
@@ -94,7 +94,7 @@ const MapEvents = () => {
       }
 
       setCoordinates(localCoordinates, eventType);
-      router.push({
+      router.replace({
```

Comment:
@denizsult Neden `replace`in fix ettigine dair bir teorin var mi?

Reply:
@denizsult shallow deneyebilir miyiz? https://nextjs.org/docs/routing/shallow-routing

shallow urli degistirip tekrar render/fetch tetiklenmemesi icin var.

Reply:
Burada https://github.com/vercel/next.js/discussions/18072#discussioncomment-107848 shallow routing'in de işe yaramadığı yazılmış. 

https://github.com/vercel/next.js/discussions/18072#discussioncomment-109059 Bu çözüm önerilmiş.

Reply:
Local'de denedim shallow uri'de re-render. [Burada](https://github.com/vercel/next.js/discussions/18072#discussioncomment-107823) da belirtilmiş. Freeze sorunu devam etti. Ancak window.history.replaceState() sorunu çözüyor. https://github.com/vercel/next.js/discussions/18072#discussioncomment-109059.